torch_xla/core/dynamo_bridge.py

```diff
@@ -529,6 +583,10 @@ def __init__(self, module):
     self._unsupported_nodes = []
 
   def run_node(self, n: torch.fx.Node):
+    # We need to restore this metric count later, so save it in a separate variable
+    dynamo_extract_graph_helper_metric_count = metrics.counter_value(
+        'DynamoExtractCompiledGraph')
```

Comment:
will `run_node` call `extract_compiled_graph` too?

Reply:
It's hard to see from documentations. However, when I try comparing metrics before/after `run_node`, from what I can see, it's not calling `extract_compiled_graph`.

Reply:
ok then I am confused what this `dynamo_extract_graph_helper_metric_count ` is doing here

Reply:
This code (`run_node`) is executed when we're fetching the fallback ops. And in this code below, we clear our metric counters via `metrics.clear_counters()`. So we need a way to restore this counter, so we can verify `extract_compiled_graph` only gets called once in our unit tests.

Reply:
Ah I see, I can fix it later. I think the right thing to do is to define a region where counter does not incremented.